.eslintrc.json

```
@@ -157,6 +157,37 @@
       "rules": {
         "filenames-simple/naming-convention": "off"
       }
+    },
+    {
+      "files": ["**/api/src/**/*.ts"],
+      "plugins": ["eslint-plugin-jsdoc"],
+      "extends": ["plugin:jsdoc/recommended"],
```

```suggestion
      "extends": ["plugin:jsdoc/recommended-typescript-error"],
```
This seems like a better default, since it disallows parameter types (which would conflict with TS)